[2024-04-20T15:28:53.938+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-20T15:28:53.960+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2023-09-13T00:00:00+00:00 [queued]>
[2024-04-20T15:28:53.972+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2023-09-13T00:00:00+00:00 [queued]>
[2024-04-20T15:28:53.974+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-20T15:28:54.005+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): first_task> on 2023-09-13 00:00:00+00:00
[2024-04-20T15:28:54.012+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=745) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-20T15:28:54.015+0000] {standard_task_runner.py:63} INFO - Started process 751 to run task
[2024-04-20T15:28:54.013+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag', 'first_task', 'scheduled__2023-09-13T00:00:00+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpsbzzsnm_']
[2024-04-20T15:28:54.016+0000] {standard_task_runner.py:91} INFO - Job 48: Subtask first_task
[2024-04-20T15:28:54.057+0000] {task_command.py:426} INFO - Running <TaskInstance: our_first_dag.first_task scheduled__2023-09-13T00:00:00+00:00 [running]> on host e87e2f3305bd
[2024-04-20T15:28:54.139+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='julionevadod' AIRFLOW_CTX_DAG_ID='our_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2023-09-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-13T00:00:00+00:00'
[2024-04-20T15:28:54.141+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-20T15:28:54.155+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-20T15:28:54.156+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2024-04-20T15:28:54.165+0000] {subprocess.py:86} INFO - Output:
[2024-04-20T15:28:54.166+0000] {subprocess.py:93} INFO - hello world, this is the first task!
[2024-04-20T15:28:54.166+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-20T15:28:54.167+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-20T15:28:54.187+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=our_first_dag, task_id=first_task, execution_date=20230913T000000, start_date=20240420T152853, end_date=20240420T152854
[2024-04-20T15:28:54.238+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-20T15:28:54.248+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-20T15:28:54.250+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
